clearcut_detection_backend/frontend/src/components/Sidebar/style.css

```diff
@@ -0,0 +1,63 @@
+.sidebar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  flex: 1 0 auto;
+  width: 337px;
+  background-color: inherit;
+  z-index: 100;
+}
+
+.sidebar-header {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.5);
+  padding: 15px 33px 30px;
+}
+
+.sidebar-body {
+  height: 100vh;
+  padding: 0 33px;
+}
```

Comment:
need to handle viewport height, because scroll is disabled in this case. see pic
![image](https://user-images.githubusercontent.com/23707448/68470371-9891ed80-0224-11ea-840e-7cd207ab101f.png)